internal/controllers/v2controllers/controller_set.go

```diff
@@ -0,0 +1,25 @@
+package v2controllers
+
+import "github.com/broadinstitute/sherlock/internal/models/v2models"
```

Comment:
Unused import?

Reply:
I don't think so, the ControllerSet constructor accepts a v2models.StoreSet that it passes to the internal controller constructors